kc-cloudtrust-testsuite/pom.xml

```diff
@@ -39,7 +39,7 @@
         <slf4j.version>1.7.36</slf4j.version>
         <greenmail.version>1.6.8</greenmail.version>
         <systemlambda.version>1.2.1</systemlambda.version>
-        <selenium.version>4.1.2</selenium.version>
+        <selenium.version>4.1.1</selenium.version>
```

Comment:
Strange

Reply:
If I leave this version or increase op to 4.1.4, then I have a NoSuchMethodError in Guava (ImmutableMap.of(....[12 parameters])...